### ISSUE TITLE
Problem Installing CVXPY with PIP

### ISSUE THREAD

**Issue by rajat-talak** [cvxpy/cvxpy#1589]
**Describe the bug**
I crate a new virtual environment using "virtualenv -p python3 env-name" and then pip install cvxpy. All works fine. However, when I type the code "import cvxpy" I get the following error:

ImportError: "..../env-name/lib/python3.8/site-packages/_scs_direct.cpython-38-x86_64-linux-gnu.so": undefined symbol: dgesv_ 

**To Reproduce**
A [minimal code example](https://stackoverflow.com/help/mcve) that reproduces the bug or unexpected behavior.
Just stated above. I see the error only with "import cvxpy"

**Expected behavior**

**Output**
If applicable, include program output. If reporting a program crash, please include the entire stack trace.

**Version**
 - OS: Ubuntu 20.04 
 - CVXPY Version: 1.1.17

**Additional context**

**Comment by phschiele** [comment]
@rajat-talak The problem seems to be that a function of [LAPACK](http://www.netlib.org/lapack/explore-html/d7/d3b/group__double_g_esolve_ga5ee879032a8365897c3ba91e3dc8d512.html) is not found.

Could you try to install:
`sudo apt-get install libopenblas-dev`

**Comment by rajat-talak** [comment]
Thanks for the reply. 

I installed it and the same problem persists. I even tried to installing:
```bash
sudo apt-get install libblas-dev liblapack-dev
```
and the problem does not go away. 

Just a correction, I get the error "undefined symbol: dgesv_" and not "undefined symbol: dgesv". I don't know if this is a problem.

Also, I think scipy also depends on LAPACK. I tried installing it separately via pip in a virtual environment. It does not give me any problems.
